### PR TITLE
Adds mupen64 hdpack folder

### PR DESCRIPTION
**Pull request by DylanTackoor**
Adds an `$emulationPath/hdpacks/retroarch/Mupen64plus/hires_texture` folder + sets related settings. Opening as a draft bc while this config looks correct to me, I haven't gotten [these textures](https://www.youtube.com/watch?v=fO0Hqhk7qxA) to load yet 🙃

### CODE REVIEW

**Comment by dragoonDorise** [comment]
Hi @DylanTackoor , we will merge this for sure as soon as you get it to work :)

Thanks!

**Comment by DylanTackoor** [comment]
I'm gonna try getting it to work with https://github.com/AndratVA/Mario-Kart-64-HD by this weekend 🤔

**Comment by DylanTackoor** [comment]
Afaik:
- `.png` textures stored in `hires_texture/` should generate a cache when loading a matching game and then work
- `.htc`/`.hts` textures in `cache/` should work but the RA settings need to match the settings used when generating the cache

But I couldn't get any textures to load or cache with the RA flatpak yesterday 🙃 I unsuccessfully tried:

- the [settings in the RA docs](https://docs.libretro.com/library/mupen64plus/#hires-textures:~:text=%2D%20mupen64plus%2DEnableTextureCache%20%3D%20%22True%22%0A%2D%20mupen64plus%2DtxHiresEnable%20%3D%20%22True%22%0A%2D%20mupen64plus%2DtxCacheCompression%20%3D%20%22True%22%0A%2D%20mupen64plus%2DtxHiresFullAlphaChannel%20%3D%20%22False%22%0A%2D%20mupen64plus%2DEnableEnhancedTextureStorage%20%3D%20%22False%22%0A%2D%20mupen64plus%2DEnableEnhancedHighResStorage%20%3D%20%22False%22) with [some of the texture packs it mentioned](https://emulationking.com/nintendo64/#:~:text=edge%20in%20performance.-,N64,-Texture%20Packs)
- [the texture pack & matching settings from this video](https://www.youtube.com/watch?v=LlKHEh-j3sg)
- a couple of the texture packs from [this Windows installer](https://github.com/Rosalie241/BetterMajorasMaskInstaller) that did work on Windows, but not when I copied the textures+settings to the flatpak
- manually just flipping different combinations of the texture settings

I'm gonna try again later this week to see if I missed something/made a typo somewhere/got a path wrong/whatever else

**Comment by Godsbane** [comment]
I tagged you in a post in the discord about mupen hd textures. Try the settings there?

i think the missing bit might be` mupen64plus-EnableTextureCache = "True"`

File: functions/EmuScripts/emuDeckRetroArch.sh [hunk 1]
```diff
@@ -1024,14 +1022,31 @@ RetroArch_Mupen64Plus_Next_setUpCoreOpt(){
...
+	RetroArch_setOverride 'Mupen64Plus-Next.opt' 'Mupen64Plus-Next'  'mupen64plus-txHiresFullAlphaChannel' '"True"'
+	RetroArch_setOverride 'Mupen64Plus-Next.opt' 'Mupen64Plus-Next'  'mupen64plus-txCacheCompression' '"True"'
+	RetroArch_setOverride 'Mupen64Plus-Next.opt' 'Mupen64Plus-Next'  'mupen64plus-EnableEnhancedHighResStorage' '"True"'
+	RetroArch_setOverride 'Mupen64Plus-Next.opt' 'Mupen64Plus-Next'  'mupen64plus-EnableEnhancedTextureStorage' '"False"' # lazy loading
+}
+
+#  setupHdPacks()
+RetroArch_Mupen64Plus_Next_setUpHdPacks(){
+  local texturePackPath="$HOME/.var/app/org.libretro.RetroArch/config/retroarch/system/Mupen64plus/hires_texture"
+	local textureCachePath="$HOME/.var/app/org.libretro.RetroArch/config/retroarch/system/Mupen64plus/cache"
```
Comment (DylanTackoor) [inline functions/EmuScripts/emuDeckRetroArch.sh:1042]: This was the problem! I should've pointed at the `Emulation/bios/Mupen64plus/` folder 🤦

**Review by dragoonDorise — APPROVED** [review]

**Pull request merged**